pkg/controller/nodes/task/handler.go

```diff
@@ -204,28 +204,49 @@ func (t *Handler) Setup(ctx context.Context, sCtx handler.SetupContext) error {
 		return err
 	}
 
+	// Not every task type will have a default plugin specified in the flytepropeller config.
+	// That's fine, we resort to using the plugins' static RegisteredTaskTypes as a fallback further below.
+	fallbackTaskHandlerMap := make(map[string]map[string]pluginCore.Plugin)
```

Comment:
nit: use those alias types for the `strings` here to clarify what they refer to

Reply:
done